GooglePlacesAutocomplete.js

```diff
@@ -348,7 +354,7 @@ export default class GooglePlacesAutocomplete extends Component {
   }
 
   _getPredefinedPlace = (rowData) => {
-    if (rowData.isPredefinedPlace !== true) {
+    if (rowData.isPredefinedPlace !== true || rowData.description == null) {
```

Comment:
I thought about this a bit.  I think we might be better off filtering out the predefined places where the description is null or length === 0.  If the description is blank, there is still a space (that was pressable) in the list.

Reply:
I would add `      res = [
        ...this.props.predefinedPlaces.filter(
          (place) => place.description && place.description.length,
        ),
      ];`
in line 101, or something similar.  It doesn't look like I can edit this PR, so let me know what you think about this and let's get this merged in.